src/Releases.ts

```diff
@@ -88,7 +91,8 @@ export class GithubReleases implements Releases {
             prerelease: prerelease,
             repo: this.inputs.repo,
             target_commitish: commitHash,
-            tag_name: tag
+            tag_name: tag,
+            make_latest: makeLatest
```

Comment:
Hmm should this be a string? The documentation indicates this should be a string with 3 values: `true`, `false` & `legacy`.

Reply:
I think you're right but it looks like `make_latest` is not yet included in the package that provides these types (which I believe is [`@octokit/plugin-rest-endpoint-methods`](https://www.npmjs.com/package/@octokit/plugin-rest-endpoint-methods)?). I tried re-installing [`@actions/github`](https://www.npmjs.com/package/@actions/github) locally, but I still didn't get type hints for this field (currently used is v5.13.0, after re-install I got v5.16.2, but latest is v6.7.0)...